Source/Diagnostics/MultiDiagnostics.H

```diff
@@ -2,9 +2,10 @@
 #define WARPX_MULTIDIAGNOSTICS_H_
 
 #include "FullDiagnostics.H"
+#include "BTDiagnostics.H"
 
 /** All types of diagnostics. */
-enum struct DiagTypes {Full};
+enum struct DiagTypes {Full, BackTransformed};
```

Comment:
Does this only include Full diagnostics and derived? Just wondering about reduced diagnostics.

We can potentially also unify that enum with the string mapping in `m_format` members (e.g. we add helper functions or members for enum <-> string here).

Reply:
We discussed this offline. Full and BT diagnostics are handled independently by the object 'multi_diags' compared to ReducedDiags 'reduced_diags'. So for now, we are not merging them.